graspologic/embed/n2v.py

```diff
@@ -92,7 +97,11 @@ def node2vec_embed(
         interpolate_walk_lengths_by_node_degree,
     )
 
-    node2vec_graph = _Node2VecGraph(graph, return_hyperparameter, inout_hyperparameter)
+    random_state = np.random.RandomState(seed=random_seed)
```

Comment:
Do we need to do anything special if random_seed is `None`?  Or does randomstate cover us?

Reply:
If it's `None`, we don't have to do anything. `RandomState` does the work AFAIK.

Reply:
https://numpy.org/doc/1.14/reference/generated/numpy.random.RandomState.html Yep, None is valid!